src/GovUk.Education.ExploreEducationStatistics.Common.Tests/GovUk.Education.ExploreEducationStatistics.Common.Tests.csproj

```diff
@@ -21,6 +22,8 @@
 
     <ItemGroup>
       <ProjectReference Include="..\GovUk.Education.ExploreEducationStatistics.Common\GovUk.Education.ExploreEducationStatistics.Common.csproj" />
+      <ProjectReference Include="..\GovUk.Education.ExploreEducationStatistics.Content.Model\GovUk.Education.ExploreEducationStatistics.Content.Model.csproj" />
+      <ProjectReference Include="..\GovUk.Education.ExploreEducationStatistics.Data.Model\GovUk.Education.ExploreEducationStatistics.Data.Model.csproj" />
```

Comment:
See comment I left on moving the Db Utils classes as not sure I follow the reason for doing that, and these two dependencies mean that say you created an Azure function project that has nothing to do with the database, content, or statistical models, and wrote tests for it that want to depend on `Common.Tests`, your now pulling all of these dependencies in.